server/cmwell-data-tools/src/main/scala/cmwell/tools/data/utils/logging/DataToolsLogging.scala

```diff
@@ -26,9 +26,11 @@ trait DataToolsLogging {
   private[data] lazy val badDataLogger = LoggerFactory.getLogger("bad-data")
 
   val label: Option[String] = None
+  val baseUrl: String = ""
+
 
   protected lazy val logger: Logger = {
-    val loggerName = if (label.isEmpty) getClass.getName else s"${getClass.getName} [${label.get}]"
+    val loggerName = if (label.isEmpty) getClass.getName else s"${getClass.getName} [${label.get} ${baseUrl}]"
```

Comment:
According to Logback convention the logger is the module name.
I know it's not the case in STP but it shouldn't be worse than it is now, even better the current implementation should be changed.